docs/application-layer/try-service-entry.md

```diff
@@ -0,0 +1,229 @@
+---
+sidebar_position: 5
+title: Try Service Entry
+---
+
+A Service Entry enables you to add entries to Istio's internal service registry so that services in the mesh can access and route to these manually specified services. This guide shows you how to configure external service access using Service Entry.
+
+## Preparation
+
+1. **Make default namespace managed by Kmesh**
+2. **Deploy Httpbin as sample application and Sleep as curl client**
+3. **Install waypoint for default namespace**
+
+   _The above steps could refer to [Install Waypoint | Kmesh](/docs/application-layer/install_waypoint.md#preparation)_
+
+## Deploy Sample Applications
+
+We need to deploy Httpbin as the target service and Sleep as the client:
+
+```bash
+kubectl apply -f ./samples/httpbin/httpbin.yaml
+kubectl apply -f ./samples/sleep/sleep.yaml
+```
+
+Check the deployment status:
+
+```bash
+kubectl get pods
+```
+
+You should see httpbin and sleep running:
+
+```bash
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-6f4464f6c5-h9x2p   1/1     Running   0          30s
+sleep-9454cc476-86vgb      1/1     Running   0          5m
+```
+
+## Configure Service Entry and Routing Rules
+
+Now we will create a Service Entry to define an external service and configure a VirtualService to route traffic to the internal service.
+
+Apply the following configuration:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: external-fake-svc
+  namespace: default
+spec:
+  exportTo:
+    - "*"
+  hosts:
+    - kmesh-fake.com
```

Comment:
since you are writing the doc, would prefer using ServiceEntry to access external service.

So you can make `kmesh-fake.com` access https://httpbin.org/